index.js

```diff
@@ -0,0 +1,18 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const menu = document.getElementById('menu-');
+  const button = document.getElementById('mob-barss');
+  const closebtn = document.getElementById('closed');
+  const menuitems = document.getElementById('mlist');
+
+  button.onclick = function showMenu() {
+    menu.style.display = 'block';
+  };
+
+  closebtn.onclick = function hideMenu() {
+    menu.style.display = 'none';
+  };
+
+  menuitems.onclick = function hideMenu() {
+    menu.style.display = 'none';
+  };
+});
```

Comment:
- [ ] **README:** I think it would be better if you fix the broken `Live Demo` link in your README file to make it more professional as it shows a **404** error page once clicked. For a short refresh about deploying a project on GitHub pages, just r**[efer this link.](https://www.youtube.com/watch/8hrJ4oN1u_8)**.
![Screenshot from 2023-04-10 12-44-06](https://user-images.githubusercontent.com/83042742/230949366-ad1470d2-4c8e-483b-83e2-49ee83e3c8fd.png)